vault-interface/src/generated/capacity.ts

```diff
@@ -11,6 +11,12 @@ import { Empty } from "./google/protobuf/empty";
 
 export const protobufPackage = "livestack";
 
+export interface CapacityInfo {
+  projectUuid: string;
+  instanceId: any;
+  totalCapacity: any; // 根据实际类型替换 'any'，比如 number
```

Comment:
Can this typing be more specific? In general we avoid using `any` unless we run out of options.

Reply:
fix in new commit, not make new pr yet